src/index.css

```diff
@@ -1,4 +1,20 @@
 @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap');
 @tailwind base;
 @tailwind components;
-@tailwind utilities;
\ No newline at end of file
+@tailwind utilities;
+
+.container {
```

Comment:
this class here will override a very important and responsive class in tailwind called container too. 

Please try to use tailwind at all times instead of creating classes. Everything here can be done using tailwind alone.